database/seeders/RoleSeeder.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Database\Seeders;
+
+use Illuminate\Database\Seeder;
+use Illuminate\Support\Facades\DB;
+
+class RoleSeeder extends Seeder
```

Comment:
mhm tutaj imo jest problem bo nie wiem czy lepszym rozwiązaniem nie jest przeniesienie takich rzeczy do migracji ale z drugiej strony jak wywalimy jakąś role to trzeba by grzebać w tej migracji albo dodawać nową migrację która usunie tę role a jeżeli będziemy używać tu constów to będzie kolejny problem... jeszcze muszę się zastanowić

Reply:
Może zostać moim zdaniem. Spokojnie z migracji da się wywołać tego seedera i ewentualnie można go wyrzucić z DatabaseSeeder, w sumie to nawet będzie trzeba, jak będzie w migracji. https://owenconti.com/posts/calling-laravel-seeders-from-migrations/.